stdlib/tkinter/ttk.pyi

```diff
@@ -207,9 +207,9 @@ class Entry(Widget, tkinter.Entry):
     ) -> dict[str, tuple[str, str, str, Any, Any]] | None: ...
     @overload
     def config(self, cnf: str) -> tuple[str, str, str, Any, Any]: ...
-    def bbox(self, index): ...
-    def identify(self, x, y): ...
-    def validate(self): ...
+    def bbox(self, index) -> tuple[int, int, int, int]: ...
+    def identify(self, x: int | str, y: int | str) -> str: ...
+    def validate(self) -> bool: ...
```

Comment:
Maybe these should be just deleted, so that they are inherited from `tkinter.ttk.Widget` and `tkinter.Entry` with the correct types.

Reply:
Do you mean delete the type hint, or delete the entire line from the stub?

Reply:
I meant deleting the whole lines.